src/lib/util/domains/domains.test.ts

```diff
@@ -0,0 +1,56 @@
+//- Utils Imports
+import { getDomainId, truncateAddress } from './domains';
+
+//- Mocks Imports
+import { rootDomainId, wilderDomainId } from './mocks';
+
+//////////////////////
+// f:: getDomainId  //
+//////////////////////
+
+describe('getDomainId', () => {
```

Comment:
Can we also test this with some problematic characters like `\`?

Reply:
Not entirely sure how to test this as ```getDomainId``` will now only take a string as I've removed the optional. If a character such as ```\``` is passed in as a string, a hash will still be returned.